api/v0/virtual_machines/virtual_machines.proto

```diff
@@ -0,0 +1,155 @@
+ *                |  ██║  ██║╚██████╔╝██║  ██║██║  ██║███████╗ |              *
+ *                |  ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝ |              *
+ *                +--------------------------------------------+              *
+ *                                                                            *
+ *                         Distributed Systems Runtime                        *
+ *                                                                            *
+ * -------------------------------------------------------------------------- *
+ *                                                                            *
+ *   Licensed under the Apache License, Version 2.0 (the "License");          *
+ *   you may not use this file except in compliance with the License.         *
+ *   You may obtain a copy of the License at                                  *
+ *                                                                            *
+ *       http://www.apache.org/licenses/LICENSE-2.0                           *
+ *                                                                            *
+ *   Unless required by applicable law or agreed to in writing, software      *
+ *   distributed under the License is distributed on an "AS IS" BASIS,        *
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ *   See the License for the specific language governing permissions and      *
+ *   limitations under the License.                                           *
+ *                                                                            *
+\* -------------------------------------------------------------------------- */
+
+syntax = "proto3";
+
+package aurae.virtual_machines.v0;
+
+option go_package = "github.com/aurae-runtime/ae/client/pkg/api/v0/virtual_machines;virtual_machinesv0";
+
+service VmService {
+  /// Reserve requested system resources for a new VM.
+  rpc Allocate(VmServiceAllocateRequest) returns (VmServiceAllocateResponse) {}
+
+  /// Free up previously requested resources for an existing VM
+  rpc Free(VmServiceFreeRequest) returns (VmServiceFreeResponse) {}
+
+  /// Start a new VM.
+  rpc Start(VmServiceStartRequest) returns (VmServiceStartResponse) {}
+
+  /// Stop one or more VMs.
+  rpc Stop(VmServiceStopRequest) returns (VmServiceStopResponse) {}
+}
+
+message VmServiceAllocateRequest{
+  VirtualMachine machine = 1;
+}
+
+message VmServiceAllocateResponse{
+  string vm_id = 1;
+}
+
+message VmServiceFreeRequest{}
+message VmServiceFreeResponse{
+  string vm_id = 1;
+}
+
+message VmServiceStartRequest{
+  string vm_id = 1;
+  string executable = 2;
+}
+message VmServiceStartResponse{}
+
+message VmServiceStopRequest{
+  string vm_id = 1;
+}
+message VmServiceStopResponse{}
+
+
+/// An Aurae virtual machine
+message VirtualMachine {
+  // The identifier of the VM
+  string id = 1;
+  
+  /// The memory size of VM
```

Comment:
i think these should be "//" not "///". it's proto, not rustdoc.  sorry for the nit.

Reply:
I was definitely rustdoc-ing here, thanks!